.github/actions/rust-build-release/tests/test_manifest_path.py

```
@@ -285,6 +287,47 @@ def test_main_errors_when_manifest_missing(
+        _rustup_exec: str, toolchain_arg: str, target_arg: str
+    ) -> tuple[str, list[str]]:
+        captured["toolchain"] = toolchain_arg
+        captured["target"] = target_arg
+        return toolchain_arg, [toolchain_arg]
+
+    harness.patch_attr("_resolve_toolchain", fake_resolve_toolchain)
+    decision = cross_decision_factory(main_module, use_cross=False)
+    harness.patch_attr("_decide_cross_usage", lambda *_, **__: decision)
+
+    def fake_cargo(
+        _spec: str, target_arg: str, manifest_arg: Path, features_arg: str
```

Ruff (0.15.10)

[warning] Unused function argument: `target_arg`

(ARG001)